app/javascript/components/applications/ApplicationDetailsContentCard.js

```diff
@@ -38,16 +38,16 @@ var generateContent = (dataCollection, field, labelMapper, i) => {
   )
 }
 
-var generateData = (label, value) => {
-  return (
-    <div className="margin-bottom--half" key={i}>
-      <h4 className="t-sans t-small t-bold no-margin">
-        {label}
-      </h4>
-      <p>{value}</p>
-    </div>
-  )
-}
+// var generateData = (label, value) => {
+//   return (
+//     <div className="margin-bottom--half" key={i}>
+//       <h4 className="t-sans t-small t-bold no-margin">
+//         {label}
+//       </h4>
+//       <p>{value}</p>
+//     </div>
+//   )
+// }
```

Comment:
should we removed this commented code?

Reply:
not even sure why it was commented out in the first place! we don't use this function anywhere so yes, will remove this commented code